lib/railsdock/templates/install/default/entrypoint.sh

```diff
@@ -63,5 +63,8 @@ then
   if [[ "$4" = "s" || "$4" = "server" ]]; then rm -rf /app/tmp/pids/server.pid; fi
 fi
 
+unset BUNDLE_PATH
```

Comment:
@nvick after our experimenting today, do you believe that these `unsets's are ineffective?

Reply:
Yeah, I believe they are ineffective.

Reply:
Another PR removing `GEM_PATH` and the `unset`